src/Sarif.Driver/Sdk/ExtensionMethods.cs

```diff
@@ -84,7 +84,7 @@ public static string FormatForVisualStudio(this Result result, IRuleDescriptor r
                 var lastComponent = components.Last();
                 messageLines.Add(
                     string.Format(
-                        CultureInfo.InvariantCulture, "{0}:{1} {2} {3}: {4}",
+                        CultureInfo.InvariantCulture, "{0}{1}: {2} {3}: {4}",
```

Comment:
Fixed bug in implementation. Now I can run the validator from the VS Tools menu, double-click on an output line, and navigate to the site of the issue.